tests/services/test_strategy_marketplace_integration.py

```
@@ -0,0 +1,209 @@
+)
+from app.services.strategy_submission_service import (  # noqa: E402
+    StrategySubmissionService,
+)
+from app.models.copy_trading import StrategyPublisher  # noqa: E402
+from app.models.trading import TradingStrategy  # noqa: E402
+from app.models.tenant import Tenant  # noqa: E402
+from sqlalchemy.dialects.sqlite.base import SQLiteTypeCompiler  # noqa: E402
+
+
+if not hasattr(SQLiteTypeCompiler, "visit_UUID"):
+    def _visit_uuid(self, type_, **_kw):  # pragma: no cover - sqlite shim
```

Ruff (0.13.1)

Unused function argument: `self`

(ARG001)